packages/flutter_tools/templates/package_ffi/go/go.mod.tmpl

```diff
@@ -0,0 +1,16 @@
+module {{projectName}}
+
+go 1.20
+
+require (
+	github.com/iancoleman/strcase v0.3.0
+	github.com/spf13/cobra v1.8.0
+	golang.org/x/tools v0.16.1
+)
+
+require (
+	github.com/inconshreveable/mousetrap v1.1.0 // indirect
+	github.com/spf13/pflag v1.0.5 // indirect
+	golang.org/x/mod v0.14.0 // indirect
+	github.com/csnewman/flutter-go-bridge v0.2.0 // indirect
```

Comment:
I wonder the best way of ensuring this stays up to date

Reply:
How is versioning handled for other templates?

Reply:
For Dart dependencies in flutter_tools, they are autorolled. But Dart dependencies in the templates are not I believe.

For non-Dart dependencies, I have no idea. cc @christopherfujino

Reply:
as far as i know, there is no auto-rolling of template deps :'(